src/build.d

```diff
@@ -1237,6 +1237,7 @@ Params:
 */
 auto run(T)(T args)
 {
+    args = remove!(`a == ""`)(args);
```

Comment:
String lambdas are on their way out and will probably be deprecated in the near future.
Also, you probably don't want to mutate the original range:


```suggestion
    args = args.filter!(a => !a.empty).array;
```

Though of course it would be nicer to fix the actual cause for this empty argument.

Reply:
The actual cause is the `env["PIC_FLAG"]` environment variable.  It is the empty string on Windows. 

I don't think adding an empty string to the DFLAGS should result in an error when `run` is executed, so this fix seems logical to me.  I was wondering if it might be a bug in Phobos's `process.execute` in that it doesn't ignore empty arguments.